components/authz-service/server/v2/project_test.go

```diff
@@ -371,9 +371,14 @@ func addProjectToStore(t *testing.T, store *cache.Cache, id, name string, projTy
 }
 
 func setupProjects(t *testing.T) (api.ProjectsClient, *cache.Cache, *mockEventServiceClient) {
+	cl, ca, _, mc, _ := setupProjectsAndRules(t)
```

Comment:
Why does `setupProjectsAndRules` return arguments that are never used?

Reply:
Not used for `setupProjects`. They _are_ used for `setupRules`. 🤷‍♂ We could clean this up, I suppose. But I don't find it that terrible.

Reply:
Ah, still used to thinking about private to namespace means private to file, but not true in Go. 👍